pkg/adapter/runtime_remote.go

```diff
@@ -831,3 +831,30 @@ func (r *LocalRuntime) Events(c *cliconfig.EventValues) error {
 	}
 	return nil
 }
+
+// Diff ...
+func (r *LocalRuntime) Diff(c *cliconfig.DiffValues, to string) ([]archive.Change, error) {
+	var changes []archive.Change
+	reply, err := iopodman.Diff().Call(r.Conn, to)
+	if err != nil {
+		return nil, err
+	}
+	for _, change := range reply {
+		changes = append(changes, archive.Change{Path: change.Path, Kind: stringToChangeType(change.ChangeType)})
+	}
+	return changes, nil
+}
+
+func stringToChangeType(change string) archive.ChangeType {
+	switch change {
+	case "A":
+		return archive.ChangeAdd
+	case "D":
+		return archive.ChangeDelete
+	default:
+		if change != "M" {
```

Comment:
Why not do 

case "M":
       return archive.ChangeModify
}
logrus.Errorf("'%s' is unknown archive type", change)

Reply:
because no value is returned

Reply:
```
	switch change {
	case "A":
		 return archive.ChangeAdd
	case "D":
		return archive.ChangeDelete
	default:
                 logrus.Errorf("'%s' is unknown archive type", change)
                 fallthrough
	case M:
		return archive.ChangeModify
	}
```

Reply:
Looks a lot cleaner.

Reply:
done